__tests__/app.test.js

```diff
@@ -43,57 +43,70 @@ describe("check for correct pathing", () => {
 })
 
 describe("/api/articles/:article_id", () => {
-
+    
   describe("GET/api/articles/:article_id", () => {
-    test("200: return article by id", () => {
-      return request(app)
-        .get("/api/articles/1")
-        .expect(200)
-        .then(({ body: article }) => {
-          expect(article).toHaveLength(1);
-          article.forEach((article) => {
-            expect(article).toEqual(
-              expect.objectContaining({
-                article_id: 1,
-                title: expect.any(String),
-                topic: expect.any(String),
-                author: expect.any(String),
-                body: expect.any(String),
-                created_at: expect.any(String),
-                votes: expect.any(Number),
-              })
-            );
-          });
-        });
-    });
-    describe("PATCH/api/articles/:article_id", () => {
-      test("200: return article by id with increased vote value", () => {
+      test("200: return article with comment_count", () => {
         return request(app)
-          .patch("/api/articles/1")
-          .send({ inc_votes: 10 })
+          .get("/api/articles/1")
           .expect(200)
-          .then(({ body: { article } }) => {
+          .then(({ body: [ article ] }) => {
```

Comment:
if we are only returning one article then I would not expect it to be in an array

Reply:
only works using [ ]. Tried early with Michael and it was the only way to get a singular result